src/module/auth/interface/auth.controller.ts

```diff
@@ -39,17 +39,17 @@ export class AuthController {
     return await this.authService.logOut(req.user.id);
   }
 
-  // @UseGuards(JwtGuard)
-  // @Post('session')
-  // async refreshSession(@Req() req: Request, @Res() res: Response) {
-  //   const cookie = req.headers.cookie;
-  //   if (cookie === undefined) {
-  //     throw new HttpException('Access denied', 403);
-  //   }
-  //   const httpOnlyToken: string = cookie?.split('=')[1];
-  //   const newAccessToken = await this.authService.refreshToken(httpOnlyToken);
+  @UseGuards(JwtGuard)
+  @Post('session')
+  async refreshSession(@Req() req: Request, @Res() res: Response) {
+    const cookie = req.headers.cookie;
+    if (cookie === undefined) {
+      throw new HttpException('Access denied', 403);
+    }
+    const httpOnlyToken: string = cookie?.split('=')[1].split(';')[0];
+    const newAccessToken = await this.authService.refreshToken(httpOnlyToken);
 
-  //   res.status(200);
-  //   res.json({ access_token: newAccessToken });
-  // }
+    res.status(200);
```

Comment:
Remember we are using nest now. We should not be using res i guess. ( Correct me if I'm wrong )

Reply:
When I use the return option, the tests indicate that error responses are handled, but when I need the correct response, the response time expires, indicating that the server is not responding. Should there be a way to include the @Res in the return?